src/Data/StoryItem.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Storyblok\ManagementApi\Data;
+
+use Storyblok\ManagementApi\Exceptions\StoryblokFormatException;
+
+class StoryItem extends BaseData
```

Comment:
@roberto-butti I don't get why we need this new class, if `Story` is not a proper representation of a Story object then we should fix it, this class seems like a copy of `Story` with a minimal change